src/common/Parameter.cpp

```diff
@@ -1764,9 +1764,13 @@ bool Parameter::set_value_from_string( std::string s )
       {
          ni = std::stoi(c);
       }
-      catch (std::invalid_argument const& e)
+      catch (std::invalid_argument)
       {
-         ni = val_min.i - 1;   // throw things out of range on invalid input
+         ni = val_min.i - 1;   // set value of ni out of range on invalid input
+      }
```

Comment:
You would want the const on the second exception also

Reply:
And crazily you want const std::invalid_argument &e, not std::invalid_argument const& e, which means something different. Const is hard.